auto.def

```diff
@@ -114,9 +114,10 @@ options {
 if {1} {
   # Keep sorted, please.
   foreach opt {
-    autocrypt bdb backtrace coverage doc everything fmemopen full-doc gdbm gnutls gpgme
-    gss homespool idn idn2 inotify kyotocabinet lmdb locales-fix lua mixmaster
-    nls notmuch pkgconf pgp qdbm sasl smime sqlite ssl testing tokyocabinet
+    autocrypt backtrace bdb coverage doc everything fmemopen full-doc gdbm
+    gnutls gpgme gss homespool idn idn2 inotify kyotocabinet lmdb locales-fix
+    lua mixmaster nls notmuch pgp pkgconf qdbm sasl smime sqlite ssl testing
+    tokyocabine
```

Comment:
```suggestion
    tokyocabinet
```

Reply:
Ah thanks for finding that. I just fixed it.